app/utils/validation.py

```diff
@@ -33,3 +36,15 @@ def clean_and_validate_retention_period(retention_period):
         retention_period += "s"
 
     return retention_period
+
+
+def clean_and_validate_filename(filename):
+    if "." not in filename:
+        raise ValueError("Filename must have a file extension, eg .csv")
```

Comment:
```suggestion
        raise ValueError("The file name must end with a file extension. For example, filename.csv")
```

@samuelhwilliams let’s discuss this.

Reply:
```suggestion
        raise ValueError("`filename` must end with a file extension. For example, filename.csv")
```